sale_order_pending_move_menu/hooks.py

```diff
@@ -0,0 +1,29 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from . import models
+from odoo.tools import sql
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+def pre_init_hook(env):
+    """This hook is used to link a sales order to existing pending moves
+    when the sale_order_pending_move_menu module is installed.
+    """
+    if not sql.column_exists(env.cr, "stock_move", "move_sale_order_id"):
+        _logger.info("")
+        env.cr.execute(
+            """
+            ALTER TABLE stock_move
+            ADD COLUMN IF NOT EXISTS move_sale_order_id INTEGER;
+            """
+        )
```

Comment:
Try `sql.create_column` instead of raw SQL statement, as available tool for Odoo v17, like

https://github.com/solvosci/slv-pos/blob/8cfc4ba57aff8db6e426d23ecdd539010c4ebf36/sale_order_pos_transferred/hooks.py#L12